libs/zzz/page-optimize/src/BonusStats.tsx

```diff
@@ -53,11 +52,15 @@ export function BonusStatsSection() {
     database.charOpts.setBonusStat(characterKey, newBonusStatTag(q), 0)
   const setDescription = useCallback(
     (description: string | undefined) => {
-      if (!characterKey || !description) return
-      database.charMeta.set(characterKey, { description: description })
+      database.charMeta.set(characterKey, { description })
+      setCharMetaDesc(description)
     },
     [database.charMeta, characterKey]
   )
+
+  useEffect(() => {
```

Comment:
remove this useeffect, it is updating the database which can break at transitions (changing to another character, useeffect triggers, the desc is set to new character)